doc/src/modules/parsing.rst

```diff
@@ -109,17 +109,13 @@ The currently-packaged LaTeX parser backend is partially generated with
 but to use the parser, you only need the ``antlr4`` Python package available.
 
 Depending on your package manager, you can install the right package with, for
-example, ``pip3`` (Python 3 only)::
+example, ``pip``::
 
-    $ pip3 install antlr4-python3-runtime
+    $ pip install antlr4-python3-runtime==4.10
```

Comment:
Judging by the release history I would guess that a 4.10.1 release will come out at some point:
https://pypi.org/project/antlr4-python3-runtime/#history

Should this not all for something like `4.10.*` rather than pinning `4.10`?

Reply:
We don’t know whether 4.10.1 will be compatible, and 4.10 is the only tested version.

Reply:
Is it not generally expected that `4.10.1` will be compatible with `4.10`? I'm not sure how their version scheme works.

Currently 4.10 is the only tested version but we can set CI to update to `4.10.1` as soon as it is available.

Reply:
Yes it's expected.
We are already testing https://github.com/sympy/sympy/blob/edf24253833ca153cb6d29ae54092ecebe29614c/.github/workflows/runtests.yml#L203

Reply:
If the `4.10.*` series is expected to remain compatible then I think it would be good to suggest that is what should be installed and also that is what should be checked for.